src/shared/components/FormComponents/SelectController/SelectController.tsx

```diff
@@ -132,7 +140,14 @@ export const SelectController = <T extends FieldValues>({
         }}
         data-testid={dataTestid}
       >
-        {renderGroupedOptions()}
+        {rootSelector && targetSelector ? (
+          <div className={rootSelector} style={{ overflowY: 'auto' }}>
```

Comment:
pls use Box from MaterialUi
```suggestion
          <Box className={rootSelector} sx={{ overflowY: 'auto' }}>
```